www/source/partials/docs/_using-builder-origin-keys.html.md.erb

```diff
@@ -0,0 +1,60 @@
+## <a name="origin-keys" id="origin keys" data-magellan-target="origin-keys">Origin Keys</a>
+## About origin keys
+
+Creating an origin automatically generates an origin key pair. 
+
+Origin key cryptography is asymmetric: it has a public origin key that you can distribute freely, and a private origin key that you should keep safe.
+
+Habitat Builder and Habitat Depot require access to at least the public origin key to validate packages in the origin. 
+
+Habitat Supervisors use the public origin key to verify the integrity of a downloaded Habitat artifact before running it.
+
+By default, Supervisors only run packages for which they have the public origin key, and cannot run packages without the correct origin public origin key.
+You can provide the origin public origin key to the Supervisors by pointing them to a depot that containing it, or by putting the key on disk outside of Habitat.
+
+## Creating origin keys
+
+Creating an origin, either on the Habitat Builder site or with the Habitat CLI, automatically generates two keys, a private origin key and a public origin key. You will use the private origin key to sign your packages and Supervisors will use the public origin key to verify the integrity of your packages (`.hart` files). 
```

Comment:
On builder, there are some additional things involved in creating an origin (entries in DB to track users, etc), but on the CLI there's no real concept of "creating an origin" apart from `hab origin key generate`, which generates the key pair. This language makes it sound like there's a `hab origin create`-type command, which there isn't.

Reply:
I changed `create` to `generate` for the CLI. I left the language on what all gets made when an origin is created on builder focused on keys.